src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -414,7 +414,8 @@
         {
             "Type": "ApiServer",
             "Parameters": {
-                "Scenario": "StartAPIServer"
+                "Scenario": "StartAPIServer",
+                "Role": "Server"
```

Comment:
I will approve but please see if you need this on client. Do you need the server to call client back?